src/player.js

```diff
@@ -1270,8 +1310,8 @@ export default class Player extends FakeEventTarget {
 
     this.hideTextTrack();
 
-    this._setDefaultTrack(TrackTypes.TEXT, this._getLanguage(playbackConfig.textLanguage, activeTracks.text, TrackTypes.TEXT), offTextTrack);
-    this._setDefaultTrack(TrackTypes.AUDIO, playbackConfig.audioLanguage, activeTracks.audio);
+    this._setDefaultTrack(TrackTypes.TEXT, this._playbackAttributesState.TEXT_LANG, this._getLanguage(playbackConfig.textLanguage, activeTracks.text, TrackTypes.TEXT), offTextTrack);
```

Comment:
@yairans `playbackConfig.textLanguage` is initial config and if there is `this._playbackAttributesState.TEXT_LANG` already it means that it replaced the initial config, so why pass both now? just pass this or that to the `_setDefaultTrack` method

Reply:
@OrenMe the `this._playbackAttributesState.TEXT_LANG` could be non exists in the new media. for this case we have to take the lang form config.

Reply:
The original config can be non exist as well, it doesn’t matter let this case. The point is, the only way you have a config here is if it was previously set, but is is only one, either from app config, auto selected or user selected.